examples/python/pre-filter/stack.yml

```diff
@@ -0,0 +1,20 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://veba.lab.local
+functions:
+  pre-filter:
+    lang: python3-flask
+    handler: ./pre-filter
+    image: pilue/pre-filter:latest
```

Comment:
Note: Once this PR is approved we need to change this to the VMware repo and a different container name.

Reply:
Understood - let me know if there's anything I need to do here.

Reply:
If you want, please take a look at the existing Python functions and suggest a name for the image.